src/runtime/manifest.ts

```diff
@@ -465,7 +465,8 @@ export class Manifest {
         } else {
           preamble = `Post-parse processing ${severity} caused by`;
         }
-        message = `${preamble} '${fileName}' line ${e.location.start.line}.
+        const fileNameStr = fileName ? `'${fileName}' ` : '';
```

Comment:
Just a thought, would it be clearer to report an error like this instead?

```
Post-parse processing error caused by <no filename> line 7.
```

Reply:
Split into https://github.com/PolymerLabs/arcs/pull/6616

Had them as different commits just generally try to avoid opening CLs that have such small changes (because of the overhead of copybara, the bots and code review)